### PR TITLE
add duckduckgo engine

### PR DESCRIPTION
**Pull request by ammbot**

### CODE REVIEW

**Review by szlend — APPROVED** [review]
Approved, but lets wait until we deploy the frontend before we merge this.

**Review by jurgenwerk — APPROVED** [review]
@ammbot The feature was deployed, so please deploy this as well. Instructions are in the readme.

**Pull request merged**
